src/test/java/edu/cornell/kfs/sys/util/CuXMLUnitTestUtils.java

```diff
@@ -17,15 +19,23 @@ public class CuXMLUnitTestUtils {
     private static final Logger LOG = LogManager.getLogger();
 
     public static void compareXML(File expectedXmlFile, File actualXmlFile) {
+        compareXMLWithEvaluatorors(expectedXmlFile, actualXmlFile, DifferenceEvaluators.Default);
+    }
+    
+    public static void compareXMLWithEvaluatorors(File expectedXmlFile, File actualXmlFile, DifferenceEvaluator... evaluators) {
         Diff xmlDiff = DiffBuilder.compare(expectedXmlFile)
                 .withTest(actualXmlFile)
                 .checkForIdentical()
                 .ignoreComments()
                 .ignoreWhitespace()
+                .withDifferenceEvaluator(
+                        DifferenceEvaluators.chain(
+                                DifferenceEvaluators.Default,
+                                new KualiDecimalXmlDifferenceEvaluator()))
```

Comment:
This method's `evaluators` argument doesn't appear to be actively used by this method; this spot is using a hard-coded list of evaluators instead. Did you mean to have it use the `evaluators` argument?

Reply:
oops, thanks for noticing, I missed this when I was iteratively working on this